src/pykiso/test_coordinator/test_case.py

```diff
@@ -183,6 +183,68 @@ def setUpClass(cls) -> None:
         if options.report_type == "junit":
             initialize_logging(None, options.log_level, options.report_type)
 
+    def setUp(self) -> None:
+        """Hook method for constructing the test fixture."""
+        pass
+
+    def test_run(self) -> None:
```

Comment:
This one should be removed so it won't count as a passed test if a subclass uses other test names